pkg/securitypolicy/securitypolicyenforcer.go

```diff
@@ -40,7 +40,7 @@ func init() {
 type SecurityPolicyEnforcer interface {
 	EnforceDeviceMountPolicy(target string, deviceHash string) (err error)
 	EnforceDeviceUnmountPolicy(unmountTarget string) (err error)
-	EnforceOverlayMountPolicy(containerID string, layerPaths []string, target string) (err error)
+	EnforceOverlayMountPolicy(containerID string, layerPaths []string, rootfsDir, upperDir, workDir string) (err error)
```

Comment:
Slight design question: Don't both `upperDir` and `workDir` share the same prefix? Can't we just pass in `ScratchDir` instead?

Reply:
they do at the moment, there's nothing about the design that says they have to in the future.

Reply:
not touching it anymore